### DISCUSSION TITLE
Rework of aio subsystem

### DISCUSSION THREAD

**Discussion by msteveb** [msteveb/jimtcl#264] [General]
Hi All,

I am working on a rework of the aio subsystem to remove use of stdio which really doesn't work with buffering and non-blocking sockets. No documentation yet, but implementation is on branch aio-no-stdio in case anyone would like to test it or provide feedback.

Cheers,
Steve

**Comment by msteveb** [comment]
Now pushed as #266 with documentation

**Reply by The-Markitecht** [reply]
I like this version very much, thank you.

I was looking at the docs in commit ba150e5efc611df2c8eecb37ba09924ac44b105b.  When do you guess that will be visible at https://jim.tcl.tk/fossil/doc/ ?

**Reply by msteveb** [reply]
Thanks. The replication into fossil was broken. Now fixed and updated
